test/runnable/test18976.d

```diff
@@ -0,0 +1,26 @@
+// https://issues.dlang.org/show_bug.cgi?id=18976
```

Comment:
This file should be in `compilable` not `runnable`. It would be nice to include the other case, too.